NOLF2/ClientShellDLL/ClientShellShared/InterfaceResMgr.h

```diff
@@ -71,6 +71,11 @@ class CInterfaceResMgr
 	void				ConvertScreenPos(int &x, int &y);
     LTFLOAT             GetXRatio()                         {return m_fXRatio;}
     LTFLOAT             GetYRatio()                         {return m_fYRatio;}
+    const LTFLOAT				Get4x3Ratio()						{ return 640.0f / 480.0f; }
```

Comment:
close:
> Here is the syntax of const member function in C++ language,
> `datatype function_name const();`
this goes for the other member functions too

Reply:
Made fixes in https://github.com/Katana-Steel/lithtech/pull/55/commits/9b8578ef812486cbb3e5b994bcb008b17d47d5c7